grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaServerCall.java

```diff
@@ -534,6 +531,12 @@ static HttpHeaders statusToTrailers(
                          Base64.getEncoder().encodeToString(proto.toByteArray()));
         }
 
+        if (ctx instanceof DefaultServiceRequestContext) {
```

Comment:
I forget if request context is unwrappable but if so better to use unwrap.

Reply:
It's not unwrappable so I guess just using `instanceof` is the best I can do. 😆